web/src/api/mocks/categories.ts

```diff
@@ -1,7 +1,195 @@
 import { Category } from 'api/typings';
 
 export const categories: Category[] = [
-    { id: 1, name: 'category1', metadata: { color: '#1b67b3' } },
-    { id: 2, name: 'category2', metadata: { color: '#fa0808' } },
-    { id: 3, name: 'category3', metadata: { color: '#313136' } }
+    {
+        id: '1',
+        name: 'category1',
+        metadata: { color: '#1b67b3' },
+        parent: null,
```

Comment:
It can be an issue when I send a request to get only root categories. Now I send a filter with parent is_null, but how can I do it in case of parentis []? 
`filters: [
                {
                    field: 'parent',
                    operator: Operators.IS_NULL
                }
            ]`